resources/js/components/user.vue

```diff
@@ -0,0 +1,357 @@
+					  	 	//Validate whether File is valid Excel file.
+        					 fileUpload = document.getElementById("swal-file1"),
+        					 regex = /^([a-zA-Z0-9\s_\\.\-:])+(.xls|.xlsx|.csv)$/;
+        					  if (regex.test(fileUpload.value.toLowerCase())) {
+					          	if (typeof (FileReader) != "undefined") {
+					                var reader = new FileReader();
+					 
+					                //For Browsers other than IE.
+					                if (reader.readAsBinaryString) {
+					                    reader.onload = function (e) {
+					                       $vm.uploadlist = ProcessExcel(e.target.result);
+					                    };
+					                    reader.readAsBinaryString(fileUpload.files[0]);
+					                } else {
+					                    //For IE Browser.
+					                    reader.onload = function (e) {
+					                        var data = "";
+					                        var bytes = new Uint8Array(e.target.result);
+					                        for (var i = 0; i < bytes.byteLength; i++) {
+					                            data += String.fromCharCode(bytes[i]);
+					                        }
+					                      $vm.uploadlist =  ProcessExcel(data);
+
+					                    };
+					                    reader.readAsArrayBuffer(fileUpload.files[0]);
+					                }
+					                if (!$vm.uploadlist.sucess){
+					                	Swal.showValidationMessage($vm.uploadlist.message);
+					                }
+					            } else {
+					                Swal.showValidationMessage("This browser does not support HTML5.");
+					            }
+					        } else {
+					           Swal.showValidationMessage('Error: please select a valid file (.xls,. xlsx or .csv file)');
+					        }
+
+					  	if ( faculty == "" || department == "") {					     
+					         Swal.showValidationMessage('All fields are required');
+					  	}
+					  	for (var i = 0; i < $vm.uploadlist.message.length; i++) {
+					  		$vm.listTrHtml += "<tr>";
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].matric_number+"</td>"
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].first_name+"</td>"
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].other_names+"</td>"
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].email+"</td>"
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].gender+"</td>"
+					  		$vm.listTrHtml += "<td>"+$vm.uploadlist.message[i].phone+"</td>"
+					  		$vm.listTrHtml += "</tr>";
+					     }
+					    return [
+					      faculty,
+					      department,
+					      $vm.uploadlist.message
+					      
+					    ]
+					  } 
+					}).then((result)=>{
+						if (result.value) {
+					    const answers = {faculty_id:result.value[0], user_name:result.value[1], user_code:result.value[2]}
+					    Swal.fire({
+					      title: 'click on proceed',
+					      text: 'other cancel and restart',
```

Comment:
Order or other sir?

Reply:
sorry that was grammatical error. it should be something like click cancel to restart or else cancel to restart @Auwalms